app/controlplane/cmd/main.go

```diff
@@ -233,6 +234,12 @@ func newSigningCA(_ context.Context, ca *conf.CA, logger log.Logger) (ca.Certifi
 		return fileca.NewFileCA(fileCa.GetCertPath(), fileCa.GetKeyPath(), fileCa.GetKeyPass(), false)
 	}
 
+	if ca.GetEjbcaCa() != nil {
+		ejbcaCa := ca.GetEjbcaCa()
+		_ = logger.Log(log.LevelInfo, "msg", "Keyless: EJBCA CA configured")
+		return ejbca.New(ejbcaCa.GetServerUrl(), ejbcaCa.GetKeyPath(), ejbcaCa.GetCertPath(), ejbcaCa.GetRootCaPath(), ejbcaCa.GetCertificateProfileName(), ejbcaCa.GetEndEntityProfileName(), ejbcaCa.GetCertificateAuthorityName()), nil
```

Comment:
it might make sense for consistency that `New` returns an error object (even if for now it's always nil) similarly to what fileCa.New does.